File: BlackJack.WebApi/Models/Hand.cs
using BlackJack.Models;
using System.Collections.Generic;

namespace BlackJack.WebApi.Models
{
    public class Hand
    {
        public string Name { get; set; }
        public int Total { get; set; }
        public string Player { get; set; }
        public int NumberOfTurnsPlayed { get; set; }

    }
}

File: BlackJack.TypeScript/src/models/Hand.ts


export class Hand {
    name :string;
    total :number;
    player :string;
    numberOfTurnsPlayed :number;

   constructor( name: string, total: number,  player: string, numberOfTurnsPlayed: number) {
    this.name = name;
    this.total = total;
    this.player = player;
    this.numberOfTurnsPlayed = numberOfTurnsPlayed; 
   }

}